### PR TITLE
update packages, use new eslint

### PR DESCRIPTION
**Pull request by mattbanks**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,9 +44,9 @@
   },
   "devDependencies": {
     "ava": "*",
-    "eslint": "^1.5.1",
+    "eslint": "^1.6.0",
```
Comment (shinnn) [inline package.json:47]: Not needed. https://nodesource.com/blog/semver-tilde-and-caret

File: package.json [hunk 2]
```diff
@@ -44,9 +44,9 @@
...
     "is-plain-obj": "^1.0.0",
-    "temp-write": "^1.1.2"
+    "temp-write": "^2.0.0"
```
Comment (sindresorhus) [inline package.json]: You can't just bump majors and hope it works. The API changed in this major and would need to be adjusted in the code.

Comment (shinnn) [inline package.json]: @sindresorhus There is no difference in `.sync` method between temp-write v1 and v2. https://github.com/sindresorhus/temp-write/commit/27fc22833b75fa89f8dc1b43123ebbce52665b35#diff-168726dbe96b3ce427e7fedce31bb0bc

We can safely bump temp-write to v2.0.0 without changing the code.

Comment (sindresorhus) [inline package.json]: right, forgot

**Pull request closed without merging**